examples/consumer.py

```diff
@@ -102,6 +102,9 @@ def print_assignment(consumer, partitions):
                                  (msg.topic(), msg.partition(), msg.offset(),
                                   str(msg.key())))
                 print(msg.value())
+                # Store the offset associated with msg to a local cache. Stored offsets are committed to Kafka by a background thread every AutoCommitIntervalMs
```

Comment:
It is called auto.commit.interval.ms in Python.

Also try to keep under 80 columns in width.

Reply:
Thanks, updated. Broken down the comment to multiple lines to reduce the width.